controller/src/main/java/gov/va/vro/controller/ClaimMetricsController.java

```diff
@@ -74,4 +85,24 @@ public ResponseEntity<List<ExamOrderInfoResponse>> allExamOrderInfo(Integer page
     ExamOrdersInfo examOrdersInfo = claimMetricsService.findAllExamOrderInfo(params);
     return ResponseEntity.ok(examOrdersInfo.getExamOrderInfoList());
   }
+
+  @Override
+  public ResponseEntity<HealthDataAssessment> healthEvidence(MasAutomatedClaimRequest request) {
+    log.info("Received health evidence request with collection ID {}", request.getCollectionId());
+    String correlationId = UUID.randomUUID().toString();
+    var payload =
+        MasAutomatedClaimPayload.builder()
```

Comment:
There should be a mapper, but maybe it isn't accessible from here? if not, i'd put in a tech debt ticket about it and i can take a look next week.